python/ray/serve/tests/test_persistence.py

```diff
@@ -14,9 +14,12 @@ def test_new_driver(serve_instance):
 from ray import serve
 serve.init()
 
-@serve.route("/driver")
 def driver(flask_request):
     return "OK!"
+
+serve.create_endpoint("driver", "/driver")
+serve.create_backend(driver, "driver")
+serve.split("driver", {{"driver": 1.0}})
```

Comment:
accidental double {{

Reply:
This is actually to escape the {{ because it's in a format string

Reply:
i see can you add a comment?

Reply:
oh it's merged. just make sure piggyback it on any follow up PR